e2e/features/record-application.feature

```diff
@@ -0,0 +1,273 @@
+    And within the `Applications` collection's first item, I should see `some application reason` for the `Reason of application` field
+    And within the `Applications` collection's first item, I should see `{$TODAY-5|D MMM YYYY}` for the `Date application was made` field
+    And within the `Applications` collection's first item, I should see `Granted` for the `Decision` field
+    And within the `Applications` collection's first item, I should see `some application decision reason` for the `Reason of decision` field
+    And within the `Applications` collection's first item, I should see `{$TODAY|D MMM YYYY}` for the `Date of decision` field
+    And within the `Applications` collection's first item, I should see `Completed` for the `Status` field
+
+  @record-application-error @RIA-1485
+  Scenario: Record adjourn application in wrong state
+
+    When I select the `Record an application` Next step
+    Then I am on the `Record an application` page
+    And I should see the text `The appellant or the respondent made an application. Record the decision that has been made on this application (and reasons). Once you have recorded the decision, you will need to action it if you haven’t already.`
+    And I should see the text `Attach a copy of the application email and any other communication.`
+
+    And the `Continue` button is disabled
+
+    When I add an item to the `Application email` collection
+    And within the `Application email` collection's first item, I upload `{@test.doc}` for the field without a label
+    And I click the `The legal representative` label
+    And I select `Adjourn` for the `Type of application` field
+    And I type `some application reason` for the `Reason for application` field
+    And I select `{$TODAY-3|DD-MM-YYYY}` for the `Date application was made` field
+    And I click the `Granted` label
+    And I type `some application decision reason` for the `Reasons for decision` field
+
+    Then the `Continue` button is enabled
+
+    When I click the `Continue` button
+    Then I am on the `Check your answers` page
+    And I should see `test.doc` in the `Application email` field
+    And I should see `The legal representative` in the `Application from` field
+    And I should see `Adjourn` in the `Type of application` field
+    And I should see `some application reason` in the `Reason for application` field
+    And I should see `{$TODAY-3|D MMM YYYY}` in the `Date application was made` field
+    And I should see `Granted` in the `Application decision` field
+    And I should see `some application decision reason` in the `Reasons for decision` field
+
+    When I click the `Record application` button
+    Then I should see the text `Unable to proceed because there are one or more callback Errors or Warnings`
+    And I should see an error in summary saying `You can't record application with 'Adjourn' type when case is in 'APPEAL_SUBMITTED' state`
```

Comment:
May be we can cover the "Expedite" scenario

Reply:
I added table for all three application types, where error should be visible

Reply:
👍